app/lib/cds_importer/record_inserter.rb

```diff
@@ -0,0 +1,56 @@
+class CdsImporter
+  class RecordInserter
+    class << self
+      delegate :instrument, :subscribe, to: ActiveSupport::Notifications
+
+      def destroy_cascade_record(record, mapper, filename)
```

Comment:
All 4 methods in this class take the same 'state' params (`record`, `mapper`, `filename`) - would it make more sense to store state on the instance and use instance methods in place of class methods? eg

```ruby
RecordInserter.new(record, mapper, filename).destroy_cascade_record
```

Reply:
I had debated an instance, yeah. Seems like this module should definitely be stateful.